docs_src/design/adr/0024-system-events.md

```diff
@@ -0,0 +1,102 @@
+
+- Set the Target Type to `&dtos.SystemEvent{}`when creating instance of `ApplicationService` using the [NewAppServiceWithTargetType](https://docs.edgexfoundry.org/2.2/microservices/application/ApplicationServiceAPI/#newappservicewithtargettype) factory function.
+- Write custom pipeline function that expects the new `SystemEvent` DTO and process it accordingly.  Similar to how the [ToLineProtocol](https://docs.edgexfoundry.org/2.2/microservices/application/BuiltIn/#tolineprotocol) pipeline function expects the Metric DTO.
+
+### Services/Modules impacted
+
+- Core Metadata service
+    - This service is the single point for device Add/Update/Delete and will be the producer of Device System Events.
+- Core Contracts module
+    - The new `SystemEvent` DTO will be added to this repository
+- Camera Management App Service Example
+    - Once Device System Event are implemented, the [Camera Management](https://github.com/edgexfoundry/edgex-examples/tree/main/application-services/custom/camera-management) example can be updated to consume the new Device System Events.
+- Device SDK/Service (future)
+    - Once Device System Events is implemented, the Device SDKs can switch to receiving these `Device System Events` via MesssageBus rather than the REST callbacks from Core Metadata. Beyond recognizing this future enhancement, it is out-of-scope for this ADR.
+
+
+## Considerations
+- This design approach can be used for future use cases, such as `System Events for Device Profiles` and `System Events for Device Services` when/if they are deemed needed.
+-  Another design approach that was considered is to use Support Notifications to send the System Events via REST. This would require consumers to create subscriptions to receive the Systems Events via REST to some endpoint on the consuming service. This subscription would be created using the existing Support Notifications [subscription](https://app.swaggerhub.com/apis/EdgeXFoundry1/support-notifications/2.2.0#/default/post_subscription) REST API. Likely each subscription would be for a specific Event Type. System Events would be Published by POSTing them to Support Notification [notification](https://app.swaggerhub.com/apis/EdgeXFoundry1/support-notifications/2.2.0#/default/post_notification) REST API, which would them forward them via REST POST to each service subscribed to the particular System Event. The System Event DTO would still be used, just sent via the REST. This approach is more complex, requires consumer services to have new REST endpoint(s) to receive the System Events and relies on REST rather than messaging, thus this approach was not chosen.
+
+## Decision
+This design will satisfy the  [System Events for Devices ](https://docs.edgexfoundry.org/2.3/design/ucr/0001-System-Events-for-Devices/) use case as well as most future System Event use cases
```

Comment:
```suggestion
This design will satisfy the  [System Events for Devices](https://docs.edgexfoundry.org/2.3/design/ucr/0001-System-Events-for-Devices/) use case as well as most future System Event use cases.
```

Reply:
> This is a vague statement for a decision made, in an ADR. We can't claim that without having more than 1 System Even use case.

Hence why I said `most`.

Reply:
We can't know what future use cases need to confidently state that this ADR covers what most of them need. You could say "possibly other future use cases".

Reply:
Sure, I can change it to `possibly other future use cases`